tests/test_run_app.py

```diff
@@ -497,7 +501,13 @@ def test_run_app_custom_backlog(patched_loop: Any) -> None:
 
 def test_run_app_custom_backlog_unix(patched_loop: Any) -> None:
     app = web.Application()
-    web.run_app(app, path="/tmp/tmpsock.sock", backlog=10, print=stopper(patched_loop))
+    web.run_app(
+        app,
+        path="/tmp/tmpsock.sock",
```

Comment:
Side note: ouch, this looks awfully hardcoded. It's probably a good idea to fix such hardcoded socket paths across tests (in a dedicated PR, of course).